### PR TITLE
Drop pkg/cmd/util/print

### PR DESCRIPTION
**Pull request by soltysh**
/assign @juanvallejo 
@deads2k fyi

### CODE REVIEW

**Comment by juanvallejo** [comment]
@soltysh need to update completions

File: pkg/oc/cli/newapp/newapp.go [hunk 1]
```diff
@@ -344,18 +330,22 @@ func (o *AppOptions) RunNewApp() error {
 		}
 
 		if o.Action.ShouldPrint() {
-			// TODO(juanvallejo): this needs to be fixed by updating QueryResult.List to be of type corev1.List
-			printableList := &corev1.List{
-				// this is ok because we know exactly how we want to be serialized
-				TypeMeta: metav1.TypeMeta{APIVersion: corev1.SchemeGroupVersion.String(), Kind: "List"},
+			list := &unstructured.UnstructuredList{
```
Comment (juanvallejo) [inline pkg/oc/cli/newapp/newapp.go]: Would be good to have a helper in the future to automatically generate these lists w metadata set + convert a list of corev1 objects into unstructured ones

**Comment by soltysh** [comment]
Missing bits generated, tagging based on previous approval.

**Pull request merged**